docs/src/content/docs/components/tabs.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: Tabs
+description: Learn how to create tabbed interfaces in Starlight.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+To create a tabbed interface useful to organize content by grouping similar information together, use the `<Tabs>` and `<TabItem>` components.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Tabs slot="preview">
+	<TabItem label="Stars">Sirius, Vega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymede</TabItem>
+</Tabs>
+
+</Preview>
+
+## Import
+
+```tsx
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+```
+
+## Usage
+
+Display a tabbed interface using the `<Tabs>` and `<TabItem>` components.
+Each `<TabItem>` must have a [`label`](#label) to display to users.
+
+<Preview>
+
+```mdx
+# src/content/docs/example.mdx
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs>
+	<TabItem label="Stars">Sirius, Vega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymede</TabItem>
+</Tabs>
+```
+
+<Fragment slot="markdoc">
+
+```markdoc
+# src/content/docs/example.mdoc
+
+{% tabs %}
+{% tabitem label="Stars" %}
+Sirius, Vega, Betelgeuse
+{% /tabitem %}
+
+{% tabitem label="Moons" %}
+Io, Europa, Ganymede
+{% /tabitem %}
+{% /tabs %}
+```
+
+</Fragment>
+
+<Tabs slot="preview">
+	<TabItem label="Stars">Sirius, Vega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymede</TabItem>
+</Tabs>
+
+</Preview>
+
+### Sync tabs
+
+Keep multiple tab groups synchronized by adding the [`syncKey`](#synckey) attribute.
+
+All `<Tabs>` on a page with the same `syncKey` value will display the same active label.
+This allows your reader to choose once (e.g. their operating system or package manager), and see their choice persisted across page navigations.
```

Comment:
```suggestion
This allows your reader to choose once (e.g. their operating system or package manager), and see their choice persist across page navigations.
```

Reply:
Heh, funny one. I actually think both can be correct in this context and have a slight preference for "persisted" while struggling to articulate why 😄 

It's really subtle but "see ... persist" is like an active "I am watching this thing persist all the time" vs "see ... persisted" being a slightly more passive "the thing is persisted by the website, I'll notice when I notice". Very much a _feeling_ of the language though, rather than either being grammatically incorrect.